ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml

```diff
@@ -59,23 +59,54 @@
             git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep 'infrastructure-playbooks/purge'
           on-evaluation-failure: dont-run
           steps:
-            - multijob:
-                name: 'ceph-ansible purge playbook testing'
-                condition: SUCCESSFUL
-                execution-type: PARALLEL
-                projects:
-                  - name: 'ceph-ansible-prs-luminous-ansible2.4-purge_bluestore_osds_non_container'
-                    current-parameters: true
-                  - name: 'ceph-ansible-prs-luminous-ansible2.4-purge_cluster_container'
-                    current-parameters: true
-                  - name: 'ceph-ansible-prs-luminous-ansible2.4-purge_cluster_non_container'
-                    current-parameters: true
-                  - name: 'ceph-ansible-prs-luminous-ansible2.4-purge_filestore_osds_container'
-                    current-parameters: true
-                  - name: 'ceph-ansible-prs-luminous-ansible2.4-purge_filestore_osds_non_container'
-                    current-parameters: true
-                  - name: 'ceph-ansible-prs-luminous-ansible2.4-purge_lvm_osds'
-                    current-parameters: true
+            # If PR is merging into stable-3.0 branch HEAD, run all but LVM scenarios
+            - conditional-step:
+                condition-kind: shell
+                condition-command: |
+                  #!/bin/bash
+                  git show-branch ${sha1} | grep -q $(git ls-remote https://github.com/ceph/ceph-ansible.git stable-3.0 | awk '{ print $1 }')
```

Comment:
Out of curiosity, where is the sha1 coming from?
Why don't we take `git rev-parse --abbrev-ref HEAD` and then compare it to `stable-3.0`?

Sounds easier to me.

Reply:
`${sha1}` comes from the github trigger.  It gets set to `refs/remotes/origin/pr/2811/merge` as an example.

Reply:
Example
```
$ git show-branch origin/pr/2633/merge 
[origin/pr/2633/merge] Merge e8c3745eaf18533f0871b7a629093029359db10e into 7e068d0b93ccc2d25c3a8f07454c534c8a26a758
```

Reply:
Thanks!